wavedroplet.js

```diff
@@ -116,15 +124,18 @@ d3.json('/json/' + get_query_param('key')[0], function(error, json) {
 
 function get_query_param(param) {
     var urlKeyValuePairs = {}
-    window.location.search.split("?")[1].split("&").forEach(function(d) {
-        var m = d.split("=");
+    window.location.hash.split("#")[1].split("&").forEach(function(d) {
+        var m = d.split("~");
         urlKeyValuePairs[m[0]] = m[1]
     })
+    console.log(window.location.hash, urlKeyValuePairs)
     return urlKeyValuePairs[param].split(',')
 }
 
-function to_stream_key(d) {
-    return d['ta'].replace(/:/g, '') + '_' + d['ra'].replace(/:/g, '');
+function to_stream_key(d, aliases) {
+    var from = aliases[d['ta']] || d['ta']
+    var to = aliases[d['ra']] || d['ra']
+    return from.replace(/:/g, '') + '_' + to.replace(/:/g, '');
```

Comment:
Note - this is slightly complicated because CSS selections have a limited character set: "In CSS, identifiers (including element names, classes, and IDs in selectors) can contain only the characters [a-zA-Z0-9] and ISO 10646 characters U+00A0 and higher, plus the hyphen (-) and the underscore (_); they cannot start with a digit, two hyphens, or a hyphen followed by a digit".  http://www.w3.org/TR/CSS2/syndata.html#characters

The documentation says that you're supposed to be able to use escape characters to get this to work with a wider range of characters, but D3 was still throwing syntax errors when I tried this. 

Therefore, I've updated it so that the stream id used for selection is concatenated with "---" but when it's visible to the user it's "→".  Is this a good solution?